package.json

```diff
@@ -17,6 +17,8 @@
     "cloudinary-react": "^1.3.0",
     "downloadjs": "^1.4.7",
     "focus-visible": "^5.1.0",
+    "@google/markerclusterer": "^2.0.8",
```

Comment:
Do we need the `@google/markerclusterer` dependency? I only see `@google/markerclustererplus` imported in the component.

Reply:
Nope, you're right we just need google/markerclustererplus.